client/src/cart-context/reducerFunction.js

```diff
@@ -0,0 +1,47 @@
+import * as util from "../utility/util_functions";
+const manageProductReducer = (state, action) => {
+  switch (action.type) {
+    case "SET_CART_ITEMS":
+      return { ...state, cartItems: action.products };
+    case "SET_WISHLISTED_ITEMS":
+      return { ...state, wishList: action.products };
+    case "INCREMENT":
+      return util.manageCart(state, action.item, 1);
+    case "DECREMENT":
+      return util.manageCart(state, action.item, -1);
+    case "REMOVE":
+      return util.removeItemFromCart(state, action.item);
+    case "WISHLIST":
+      return util.manageWishList(state, action.item);
+    case "sortByPrice":
+      return { ...state, sortByPrice: action.payload };
+    case "SHOWALLPRODUCTS":
+      return { ...state, showAllProducts: !state.showAllProducts };
+    case "FASTDELIVERY":
+      return { ...state, fastDelivery: !state.fastDelivery };
+    case "PRICERANGE":
```

Comment:
- Try to use a single format everywhere for naming the dispatch types.
- Generally `UPPER_CASE_WITH_UNDERSCORES` will be fine.

Reply:
yeah I have corrected it

Reply:
corrected